README.md

```diff
@@ -1,28 +1,24 @@
 # AtomicLti
-Short description and motivation.
+Atomic LTI implements the LTI Advantage specification.
 
 ## Usage
-How to use my plugin.
+Add the gem:
 
-## Installation
-Add this line to your application's Gemfile:
+  `gem 'atomic_lti', git: 'https://github.com/atomicjolt/atomic_lti.git', tag: '1.0.9'`
 
-```ruby
-gem 'atomic_lti'
-```
+Note that the gem is not open source and not available on RubyGems.org
```

Comment:
It's actually a public repo, we just didn't publish to rubygems

Reply:
I removed the comment